inst/assets/bibliography.bib

```diff
@@ -2139,3 +2139,25 @@ @article{Geistlinger2023
     url = {https://doi.org/10.1038/s41587-023-01872-y},
     year = {2023}
 }
+
+@article{Schloss2024rarefaction1,
+    author = {Patrick D. Schloss},
+    title = {Rarefaction is currently the best approach to control for uneven sequencing     effort in amplicon sequence analyses},
+    journal = {mSphere},
```

Comment:
Put this in double curly brackets to ensure that the capitalization will be maintained in rendering to other formats. Like {{mSphere}} (this is a LaTeX trick)